tools/components/admin/requests/renewal-additional-questions-form.ts

```diff
@@ -0,0 +1,6 @@
+import { Renewal } from '@lib/graphql/types'; // Renewal type
```

Comment:
Sorry, one last thing, could you please move these types to `tools/components/admin/requests/forms/types.ts`?

Reply:
Addressed!